src/components/Tearsheet/TearsheetSmall/TearsheetSmall.stories.js

```diff
@@ -172,6 +177,64 @@ storiesOf(patterns('TearsheetSmall'), module)
         isOpen
         loading={loading}
         labels={labels}
+        body={({ ...isLoading }) => (
```

Comment:
```suggestion
        body={({ isLoading }) => (
```
I believe this should apply the correct `tabIndex` for `CodeSnippet`s, as the current deploy is outputting `-1`:

![Tearsheet](https://user-images.githubusercontent.com/3846874/84502786-e2f2b580-acb0-11ea-8884-209adf37ccf6.png)